source/common/filter/tcp_proxy.h

```diff
@@ -79,6 +82,7 @@ class TcpProxyConfig {
 
   std::vector<Route> routes_;
   const TcpProxyStats stats_;
+  std::vector<Http::AccessLog::InstanceSharedPtr> access_logs_;
```

Comment:
is this for the access log system to collect the access logs? Or is this a handle to the access log handler? IF so, might be good to rename it to something appropriate or add a comment to that effect. On first read, it looks like its storing access log per connection, in a vector.

Reply:
I copied this from http_connection_manager.h, and there's no comment there either.  I think readers just need to look up the definition of the type to figure out what it is (as is the case with many things).

Reply:
It's a handle to the access log. Feel free to rename / add comments.